function Utils_ThrowError(message) {
    console.error(message);
}